.github/packer/aws-ubuntu-docker.pkr.hcl

```diff
@@ -0,0 +1,130 @@
+    owners      = ["099720109477"]
+  }
+  ssh_username = "ubuntu"
+  ami_users = []
+  ami_regions = local.all_regions
+  
+  tags = {
+    Name = "public-avalanchecli-ubuntu-focal-20.04-docker"
+    Release = "ubuntu-20.04"
+    Org = "avaplatform"
+    Base_AMI_ID = "{{ .SourceAMI }}"
+    Base_AMI_Name = "{{ .SourceAMIName }}"
+    }
+}
+
+source "amazon-ebs" "ubuntu_arm64" {
+  ami_name      = "public-avalanchecli-ubuntu-focal-20.04-docker-arm64-{{timestamp}}"
+  ami_description = "Avalanche-CLI Ubuntu 20.04 Docker"
+  instance_type = "t4g.xlarge"  # Adjust the instance type for arm64
+  region        = "us-east-1"
+  source_ami_filter {
+    filters = {
+      name                = "ubuntu/images/*ubuntu-focal-20.04-arm64-server-*"  # Filter for arm64 AMIs
+      root-device-type    = "ebs"
+      virtualization-type = "hvm"
+    }
+    most_recent = true
+    owners      = ["099720109477"]
+  }
+  ssh_username = "ubuntu"
+  ami_users = []
+  ami_regions = local.all_regions
+  tags = {
+    Name = "public-avalanchecli-ubuntu-focal-20.04-docker-arm64"
+    Release = "ubuntu-20.04"
+    Org = "avaplatform"
+    Base_AMI_ID = "{{ .SourceAMI }}"
+    Base_AMI_Name = "{{ .SourceAMIName }}"
+  }
+}
+
+
+build {
+  name    = "docker"
+  sources = [
+    "source.amazon-ebs.ubuntu_amd64",
+    "source.amazon-ebs.ubuntu_arm64",
+    "source.googlecompute.ubuntu_gcp"
+  ]
+
+    provisioner "shell" {
+        inline = [
+            "export DEBIAN_FRONTEND=noninteractive",
+            "sudo add-apt-repository -y ppa:longsleep/golang-backports",
+            "sudo apt-get -y update && sudo apt-get -y dist-upgrade && sudo apt-get -y install ca-certificates curl gcc git golang-go=2:1.22~3longsleep1",
+            "sudo install -m 0755 -d /etc/apt/keyrings && sudo curl -fsSL https://download.docker.com/linux/ubuntu/gpg -o /etc/apt/keyrings/docker.asc && sudo chmod a+r /etc/apt/keyrings/docker.asc",
+            "echo \"deb [arch=$(dpkg --print-architecture) signed-by=/etc/apt/keyrings/docker.asc] https://download.docker.com/linux/ubuntu $(. /etc/os-release && echo \"$VERSION_CODENAME\") stable\" | sudo tee /etc/apt/sources.list.d/docker.list > /dev/null",
+            "sudo apt-get -y update && sudo apt-get -y install docker-ce docker-ce-cli containerd.io docker-buildx-plugin docker-compose-plugin docker-compose",
+            "sudo usermod -aG docker ubuntu",
+            "sudo systemctl enable docker || true",
+            "sudo systemctl start docker || true",
+            "sudo chmod 666 /var/run/docker.sock",
+
+        ]
+    }
+    provisioner "shell" {
+        inline = [
+            "docker pull avaplatform/avalanchego",
+            "docker pull grafana/promtail:3.0.0",
+            "docker pull grafana/loki:3.0.0",
+            "docker pull prom/node-exporter:v1.7.0",
+            "docker pull grafana/grafana:10.4.1",
+            "docker pull prom/prometheus:v2.51.2",
+            "docker pull avaplatform/awm-relayer:v1.1.0"
```

Comment:
why not using cli docker?

Reply:
wdym?

Reply:
why not installing the CLI docker image?